add the global variables (#3530)
TYPO3-Documentation/TYPO3CMS-Reference-CoreApi: Documentation/ApiOverview/Authentication/Index.rst

:php:`\TYPO3\CMS\Frontend\Authentication\FrontendUserAuthentication`
for the frontend (FE), which both inherit from class
:php:`\TYPO3\CMS\Core\Authentication\AbstractUserAuthentication`.

> The objects for these classes are available in the global variables 

Reviewer: ```suggestion
The objects for these classes are available via
```

:php:`$GLOBALS['BE_USER']` for :php:`BackendUserAuthentication` and 

> :php:`$GLOBALS['FE_USER']->fe_user` for :php:`FrontendUserAuthentication`

Reviewer: At least the PSR-7 request attribute "frontend.user" should be used where possible:
https://docs.typo3.org/m/typo3/reference-coreapi/12.4/en-us/ApiOverview/RequestLifeCycle/RequestAttributes/FrontendUser.html
Avoid global variables where possible.

Author: I have changed it now.
